warp10/src/main/java/io/warp10/continuum/egress/EgressExecHandler.java

```diff
@@ -622,6 +624,16 @@ public void handle(String target, Request baseRequest, HttpServletRequest req, H
 
       Sensision.set(SensisionConstants.SENSISION_CLASS_WARPSCRIPT_JVM_FREEMEMORY, Sensision.EMPTY_LABELS, Runtime.getRuntime().freeMemory());
 
+      //
+      // Record FoundationDB current network thread busyness
+      //
+
+      if (storeClient instanceof FDBStoreClient) {
+        FDBStoreClient fsc = (FDBStoreClient) storeClient;
+        FDBPool pool = fsc.getPool();
+        Sensision.set(SensisionConstants.SENSISION_CLASS_WARPSCRIPT_FDB_BUSYNESS, Sensision.EMPTY_LABELS, pool.getDatabase().getMainThreadBusyness());
```

Comment:
The call to getDatabase returns an instance of Database from the pool, so you will not get a global view of busyness, only for a single Database instance among the pool. This is probably not sufficient, it would be more beneficial to add a method to FDBPool to report busyness for all pooled Database instances with a `dbinstance` label with a number from 0 to the pool size - 1.

Reply:
Indeed, I added a method into FDBPool and use labels to give data about each thread. In case you are wondering if load on this `libfdb` call can slow down the thread itself, it should'nt, the value is updated by `libfdb` itself every second. We're just reading its value.